infra/ansible-docker/roles/lakekeeper/files/bootstrap-warehouse.py

```
@@ -1,70 +1,167 @@
+
+    @property
+    def realm_url(self) -> str:
+        return self.url + f"/realms/{self.realm}"
+
+    @property
+    def realm_admin_url(self) -> str:
+        return self.url + f"/admin/realms/{self.realm}"
+
+    @property
+    def openid_config(self) -> Dict[str, Any]:
+        response = requests.get(self.realm_url + "/.well-known/openid-configuration")
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)